spec/system/task_spec.rb

```diff
@@ -58,23 +57,22 @@
   end
 
   describe 'Task編集' do
+    let!(:task) { create(:task) }
+    let!(:status_done_task) { create(:task, :done_task) }
```

Comment:
trait名に`task`はなくても良いかなと思います。
```suggestion
    let!(:done_task) { create(:task, :done) }
```

Reply:
a9e5674 で修正しました